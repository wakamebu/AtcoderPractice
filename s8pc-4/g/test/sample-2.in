7 9
-1 1
0 5
0 7
0 8
1 3
4 1
5 1
2 1 1
2 1 2
1 1 2 3
1 4 1 1
2 3 1
2 0 2
3 6 1
3 7 11
2 0 15
