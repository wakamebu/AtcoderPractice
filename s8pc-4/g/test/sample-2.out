8
9
8
31
49
